chart/templates/deployment.yaml

```diff
@@ -147,6 +147,10 @@ spec:
         # Mount rancher custom-logos volume
         - mountPath: /usr/share/rancher/ui/assets/images/logos
           name: custom-logos
+          subPath: {{ .Values.customLogos.volumeSubpaths.emberUi | default "ember" | quote }}
+        - mountPath: /usr/share/rancher/ui-dashboard/dashboard/_nuxt/img/pl
+          name: custom-logos
```

Comment:
not sure if it matters, but two names are the same.

Reply:
Same volume name but distinct subpath, https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath